src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -43,17 +56,17 @@ Sometimes, you need some setup code, or other things that would distract
 from your example, but are important to make the tests work. Consider
 an example block that looks like this:
 
-```text
-/// Some documentation.
-# fn foo() {}
-```
+   ```text
+   /// Some documentation.
+   # fn foo() {}
+   ```
```

Comment:
also it caused [a travis failure](https://travis-ci.org/rust-lang/rust/jobs/267150878):

    [01:04:47] ---- /checkout/src/doc/rustdoc/src/documentation-tests.md - Documentation_tests (line 68) stdout ----
    [01:04:47] 	error: unknown start of token: `